frontend/src/app/modules/user-card/user-card.component.html

```diff
@@ -68,17 +68,17 @@
             </button>
         </div>
         <div *ngIf="user.userFriendshipStatus === eUserFriendshipStatus.Requester" class="buttons">
-            <button class="button accept-button" (click)="onAcceptClicked()">
+            <button class="button accept-button" (click)="onAcceptClicked()" id="hover">
                 <img src="assets/user-card-icons/accept.svg" alt="message">
                 <span>Accept</span>
             </button>
-            <button class="button" (click)="onRejectClicked()">
+            <button class="button" (click)="onRejectClicked()" id="hover">
                 <img src="assets/user-card-icons/cancel.svg" alt="follow">
                 <span>Cancel</span>
             </button>
         </div>
         <div *ngIf="user.userFriendshipStatus === eUserFriendshipStatus.Friend" class="buttons">
-            <button class="button">
+            <button class="button" id="hover">
```

Comment:
Why you need id? You can use class or even assign globally excluding disabled buttons